man/man8/zpool.8

```diff
@@ -123,7 +123,7 @@
 .Cm iostat
 .Op Oo Oo Fl c Ar SCRIPT Oc Oo Fl lq Oc Oc Ns | Ns Fl rw
 .Op Fl T Sy u Ns | Ns Sy d
-.Op Fl ghHLpPvy
+.Op Fl ghHLpPvyn
```

Comment:
nit: these flags should be kept in alphabetical order, so `ghHLnpPvy`.

Reply:
Done